web/static/img/atlassian-crowd-icon.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
+1 for making the name clearer.

Mainline Dex requires that the directory structure and the template files exist (see https://github.com/dexidp/dex/blob/fc7f1ef6cc34b0ce181c964d076df57b4bf0b171/server/templates.go#L17-L35). At least one case where Dex would need to serve a UI is if a user had multiple auth providers configured. Most of the templates and directories we could probably just `touch` in the Dockerfile instead of including working templates.

We could also try and maintain a fork of Dex, they're not really amenable to the "embedded" use case so PRs that would make our lives easier probably won't make it upstream.